practice_2/WeatherApp/VIew/DetailsWeatherViewController.swift

```diff
@@ -0,0 +1,64 @@
+import UIKit
+
+class DetailsWeatherViewController: UIViewController {
+    var weather: Weather?
+    var forecastDays = [Forecast?]()
+    var unit: String = defaultUnits.unit
+
+    @IBOutlet private weak var tableView: UITableView!
```

Comment:
just a minor style remark
@IBOutlet  put it on top of  var 
to have the same code style in the whole project

Reply:
Done.
Thank you.